### ISSUE TITLE
Error displayed

### ISSUE THREAD

**Issue by FedeGueli** [GenSpectrum/LAPIS#307]
i got this error message:
![Schermata 2023-07-31 alle 09 16 19](https://github.com/GenSpectrum/LAPIS/assets/87669813/cef405e8-df07-4c23-b9ff-b62576e81430)
https://cov-spectrum.org/explore/World/AllSamples/from%3D2023-04-01%26to%3D2023-07-24/variants?nucMutations=G9652T%2CC20544T&nucMutations1=C20544T&nextcladePangoLineage1=XBB.2.3.2*&aaMutations2=orf1a%3AD48Y&nextcladePangoLineage2=XBB.2.3.2*&aaMutations3=orf1a%3AT2977I&nextcladePangoLineage3=XBB.2.3.2*&aaMutations4=orf1a%3AC7162T&nextcladePangoLineage4=XBB.2.3.2*&analysisMode=CompareEquals&

**Comment by chaoran-chen** [comment]
Thanks for reporting this, @FedeGueli! The error message is unfortunately not very precise: The problem is that ORF1a:7162T is wrong (7162 is too large) and we should return a proper error message.

**Comment by FedeGueli** [comment]
> Thanks for reporting this, @FedeGueli! The error message is unfortunately not very precise: The problem is that ORF1a:7162T is wrong (7162 is too large) and we should return a proper error message.

aahha sorry my bad!!

**Comment by fengelniederhammer** [comment]
Should we keep this bug to check what the error message looks like in LAPIS 2? And improve there, if necessary?

**Comment by chaoran-chen** [comment]
yes, that sounds like a good idea! (Once we launched LAPIS 2, we should go through all open issues and see which have been solved and which are good suggestions that we want to add.)

**Comment by chaoran-chen** [comment]
LAPIS is doing great but the issue is the cov-spectrum website. Now, it just shows the spinner if we open the link that Fede posted above:

![image](https://github.com/GenSpectrum/LAPIS/assets/18666552/1ac98c10-ac25-4c10-8c8f-30ec9892b686)

Instead, it should properly show the error message on the website.
